tools/shared.py

```diff
@@ -2994,13 +3006,22 @@ def add_emscripten_metadata(js_file, wasm_file):
       WebAssembly.lebify(EMSCRIPTEN_ABI_MAJOR) +
       WebAssembly.lebify(EMSCRIPTEN_ABI_MINOR) +
 
-      # static bump
       WebAssembly.lebify(mem_size) +
 
-      # table size
-      WebAssembly.lebify(table_size)
+      WebAssembly.lebify(table_size) +
+
+      WebAssembly.lebify(global_base) +
+
+      WebAssembly.lebify(dynamic_base) +
+
+      WebAssembly.lebify(dynamictop_ptr) +
+
+      WebAssembly.lebify(tempdouble_ptr) +
+
       # NB: more data can be appended here as long as you increase
       #     the EMSCRIPTEN_METADATA_MINOR
+
+      b''
```

Comment:
Is this needed?

Reply:
Just for symmetry purposes with the preceding lines (each ending with `+`). Fine to take out at the small expense of potentially incurring unnecessary line diffs in future edits of this function.

Reply:
Ah I see.  How about using `contents = b''.join(...` , then you can use trailing commas on each line?

Reply:
I could do that but then I'd have to change more lines of code :) (see this comment for past comment on keeping code style consistent: https://github.com/emscripten-core/emscripten/pull/7815#discussion_r245482003) I have no problem doing it if you really think it's worth it.

Reply:
OK, but if you are going to leave the multiline `+` but in that case I would drop the extra `b''`. 

Also, is there a reason why echo the lines above is separated by an empty line?   This is odd, especially since its single expression.